lib/noticed/base.rb

```diff
@@ -97,17 +97,16 @@ def run_delivery_method(delivery_method, recipient:, enqueue:)
       }
 
       run_callbacks delivery_method[:name] do
-        klass = get_class(delivery_method[:name], delivery_method[:options])
+        klass = get_class_for(delivery_method[:name], delivery_method[:options])
         enqueue ? klass.perform_later(args) : klass.perform_now(args)
       end
     end
 
-    # Retrieves the correct class for a delivery method
-    def get_class(name, options)
+    def get_class_for(delivery_method, options)
```

Comment:
Maybe we should call this `delivery_method_for(name, options)` ?

Reply:
I like it--made some changes.